error.go

```diff
@@ -3,163 +3,171 @@
 package iproto
 
 // IPROTO error code constants, generated from
-// tarantool/src/box/errcode.h
+// tarantool-master/src/box/errcode.h
 type Error int
 
 const (
 	// "Unknown error"
 	ER_UNKNOWN Error = 0
-	// "Illegal parameters, %s"
+	// "%s" "" STRING
 	ER_ILLEGAL_PARAMS Error = 1
 	// "Failed to allocate %u bytes in %s for %s"
 	ER_MEMORY_ISSUE Error = 2
-	// "Duplicate key exists in unique index \"%s\" in space \"%s\" with old tuple - %s and new tuple - %s"
+	// nolint:lll
+	// "Duplicate key exists in unique index \"%s\" in space \"%s\" with old tuple - %s and new tuple - %s" "index" STRING "space" STRING "" STRING "" STRING "old_tuple" TUPLE "new_tuple" TUPLE
 	ER_TUPLE_FOUND Error = 3
-	// "Tuple doesn't exist in index '%s' in space '%s'"
+	// Unused
 	ER_TUPLE_NOT_FOUND Error = 4
-	// "%s does not support %s"
+	// "%s does not support %s" "" STRING "" STRING
```

Comment:
Let's left only the error message here. The comments at now looks confusing and I don't see much sense in the rest of the parts.
```suggestion
	// "%s does not support %s"
```

Reply:
Ok, except that would require complicating the regexp.